src/content/en/updates/2018/03/smooshgate.md

```diff
@@ -0,0 +1,217 @@
+project_path: /web/_project.yaml
+book_path: /web/updates/_book.yaml
+description: Recently the internet was on fire because of #SmooshGate. What is it about, and where do we go from here? This write-up gives an overview.
+
+{# wf_updated_on: 2018-03-19 #}
+{# wf_published_on: 2018-03-19 #}
+{# wf_tags: javascript #}
+{# wf_featured_image: /web/updates/images/generic/info.png #}
+{# wf_featured_snippet: Recently the internet was on fire because of #SmooshGate. What is it about, and where do we go from here? This write-up gives an overview. #}
+{# wf_blink_components: Blink>JavaScript>Language #}
+
+# #SmooshGate FAQ {: .page-title }
+
+{% include "web/_shared/contributors/mathiasbynens.html" %}
+
+## What the _smoosh_ happened?! {: #what }
+
+A [proposal](https://tc39.github.io/proposal-flatMap/) for a JavaScript
+language feature called `Array.prototype.flatten` turns out to be
+Web-incompatible. Shipping the feature in Firefox Nightly caused [at least
```

Comment:
What does Webb Incompativble mean? is there a link to a definition.

Reply:
Hmm, not that I know of. The next sentence explains it though so perhaps it’s fine?

Reply:
sure. sg.